core/src/main/java/io/aiven/klaw/config/EmailConfig.java

```diff
@@ -27,7 +27,7 @@ public class EmailConfig {
   private String smtpFromPassword;
 
   @Value("${spring.mail.properties.mail.smtp.auth:null}")
-  private String smtpAuthentication;
```

Comment:
the default here is null it should probably be false?

Reply:
Updated.